.github/CONTRIBUTING.md

```diff
@@ -1,17 +1,19 @@
 
-#  Contributing to Ignite
+#  Contributing to Ignite CLI
```

Comment:
Should we change Ignite 2 to Ignite CLI on the website as well? https://infinite.red/ignite